web/wp-content/mu-plugins/alter-wpcfm-config-path.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Plugin Name: Alter-wpcfm-config-path
```

Comment:
Could the terminus project setup command ask you whether you want this feature or not?